package.json

```diff
@@ -42,6 +42,7 @@
     "lodash": "^4.17.21",
     "mathjax": "^3.2.2",
     "mathjax-full": "^3.2.2",
+    "mdast-util-find-and-replace": "^3.0.0",
```

Comment:
Considering that massive update to package-lock.json I'd suggest doing `npm ci` without the changes to `package*.json` to get back to a clean slate and then `npm i mdast-util-find-and-replace`. 

This gives me a much nicer diff (node 18.17.1 / npm 9.6.7):

```
[adrian@eluvian:~/dev/indico/py3/src:master *$]> git diff --stat
 package-lock.json | 113 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 package.json      |   1 +
 2 files changed, 114 insertions(+)
```

(also you're currently downgrading some stuff in your updated lockfile)